style.css

```diff
@@ -319,6 +628,109 @@ body {
   .scroll {
     display: none;
   }
+
+  .order {
+    padding-left: 4vw;
+    padding-right: 4vw;
+    padding-top: 6vw;
+  }
+
+  .order-images {
+    margin-right: 2.5vw;
+  }
+
+  .order-image__first {
+    margin-top: 65px;
+    height: 50%;
+    width: 65%;
+  }
+
+  .order-image__second {
+    height: 85%;
+    width: 90%;
+    margin-left: 30px;
+    margin-top: 0;
+  }
+
+  .order-details {
+    padding-top: 4vw;
+  }
+
+  .order-content {
+    margin-bottom: 3vw;
+    padding-left: 3vw;
+    padding-right: 4vw;
+    padding-top: 3.5vw;
+    width: 400px;
+  }
+
+  .order__title {
+    font-size: 34px;
+    line-height: 35px;
+  }
+
+  .order__subtitle {
+    font-size: 14px;
+    line-height: 14px;
+    margin-bottom: 3.2vw;
+  }
+
+  .order__description {
+    font-size: 13px;
+    line-height: 18px;
+    margin-bottom: 3vw;
+  }
+
+  .color-list {
+    height: 30px;
+    margin-bottom: 4.5vw;
+    margin-right: 1vw;
+  }
+
+  .list__item {
+    margin-right: 0.8vw;
+  }
+
+  .list__item_grey,
+  .list__item_blue,
+  .list__item_green {
+    width: 25.18px;
+    height: 25px;
+  }
+
+  .list__item_pink {
+    width: 30px;
+    height: 30px;
+  }
+
+  .order__price {
+    font-size: 34px;
+    line-height: 50px;
+  }
+
+  .order-number {
+  	margin-right: 1.2vw;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/LOCO_proj_Boiko/pullRequest?prid=5841754)